.gitignore

```diff
@@ -15,9 +15,23 @@ installedPackages/*
 #It is recommended that you ignore managed components that cannot be modified 
 #since there is no need to track them in Git, like for example:
 
+applications/copado__*
 classes/copado__*
+customPermissions/copado__*
+dashboards/copado__*
+email/copado__*
 triggers/copado__*
 pages/copado__*
+workflows/copado__*
+objects/copado__*
+objectTranslations/copado__*
+layouts/copado__*
```

Comment:
Some customers personalize Copado in Sandboxes and then deploy the changes using Copado. I originally included some of these lines in the gitignore but customers opened cases because Copado changes in objects/layouts did not appear when committing them to Git. To avoid these type of cases we only included metadata types that cannot be modified in Managed Packages (apex classes, etc).

Reply:
Sounds fair.
Maybe include the rest commented with a note

Reply:
Yeah that would be good!

Reply:
what about something like:

```
classes/*__*
triggers/*__*
pages/*__*
components/*__*
```